### PR TITLE
feat: `postgres` and `mysql` collection scripts

### PR DESCRIPTION
**Pull request by wpuziewicz**
First iteration of collectors for Postgres and MySQL

### CODE REVIEW

File: scripts/collector/oracle/sql/extracts/users.sql [hunk 1]
```diff
@@ -14,7 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 spool &outputdir/opdb__users__&v_tag
+<<<<<<< HEAD
+
+=======
```
Comment (cofin) [inline scripts/collector/oracle/sql/extracts/users.sql:19]:
```suggestion
<<<<<<< HEAD

=======
```
These look like merge conflicts?

File: scripts/collector/oracle/sql/extracts/users.sql [hunk 2]
```diff
@@ -14,7 +14,11 @@ See the License for the specific language governing permissions and
...
 prompt PKEY|CON_ID|USERNAME|DMA_SOURCE_ID|DMA_MANUAL_ID
+>>>>>>> main
```
Comment (cofin) [inline scripts/collector/oracle/sql/extracts/users.sql:21]:
```suggestion
>>>>>>> main
```

This one too.

File: scripts/collector/postgres/sql/op_collect.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
+\i sql/init.sql
+\i sql/applications.sql
+--\i sql/awsextensiondependency.sql
+--\i sql/awsoracleexists.sql
+--\i sql/awsoracleversion.sql
```
Comment (cofin) [inline scripts/collector/postgres/sql/op_collect.sql:5]: What's the reason for disabling these right now?

File: scripts/collector/oracle/sql/extracts/users.sql (hunk 1 above)
Comment (wpuziewicz) [reply scripts/collector/oracle/sql/extracts/users.sql:19]: Argh, must have missed it when merging in VS code.

File: scripts/collector/oracle/sql/extracts/users.sql (hunk 2 above)
Comment (wpuziewicz) [reply scripts/collector/oracle/sql/extracts/users.sql:21]: Same

File: scripts/collector/postgres/sql/op_collect.sql (hunk 3 above)
Comment (wpuziewicz) [reply scripts/collector/postgres/sql/op_collect.sql:5]: I'll have to check.

**Review by cofin — APPROVED** [review]
LGTM - Nice work

**Review by cofin — APPROVED** [review]

**Pull request merged**